### PR TITLE
updating discard mortality rate rule to use new netType and gearType

### PR DESCRIPTION
**Pull request by melinashak-noaa**

### CODE REVIEW

File: bn-models/src/models/_lookups/gear-type.ts [hunk 1]
```diff
@@ -1,29 +0,0 @@
-import { BaseLookup } from "../_base";
```
Comment (sethgerou-noaa) [inline bn-models/src/models/_lookups/gear-type.ts:1]: gear-type should not be deleted - it's still used for observer, just not for EM

**Review by sethgerou-noaa — CHANGES_REQUESTED** [review]
please re-add gear-type model + enum.  we're not using for EM, but it's still valid for observer.

File: bn-models/src/models/_lookups/gear-type.ts (hunk 1 above)
Comment (melinashak-noaa) [reply bn-models/src/models/_lookups/gear-type.ts]: good point, okay just re-added it

**Comment by melinashak-noaa** [comment]
ended up using smallFootropeTrawl and largeFootropeTrawl to represent bottomTrawl. Copying what I did previously, but if it should be represented using bottomOrRollerTrawl instead, let me know.

**Comment by sethgerou-noaa** [comment]
hmm, now that I think about it, you probably should check with Neil on what is bottom trawl in this context, could be that bottomOrRollerTrawl, smallFootRopeTrawl, and largeFootropeTrawl are all bottom trawl.  Actually, I'm not sure about any except pelagicMidwaterTrawl (not bottom trawl).

**Comment by melinashak-noaa** [comment]
okay just checked with Neil, and added bottomOrRollerTrawl, selectiveFlatfishTrawl, largeFootropeTrawl, and smallFootropeTrawl

**Pull request merged**